karate/src/test/java/api/entities/entities-create.feature

```diff
@@ -0,0 +1,58 @@
+@entities
+Feature: Test implementation of POST /entities (6.4.3.1)
+
+Background:
+  * url urlBase
+  * def buildings = read('classpath:samples/buildings.json')
+  * def buildingMinimal = read('classpath:samples/building-minimal.jsonld')
+  * def building = read('classpath:samples/building.jsonld')
+  * def buildingWithoutContext = read('classpath:samples/building-without-context.json')
+  * configure charset = null
+
+Scenario Outline: Create Buildings with different payloads
+    Given path 'entities'
+    And request __row
+    And header Content-Type = 'application/ld+json'
+    When method post
+    Then status 201
+Examples:
+    | buildings |
+
+Scenario: Create Building with unsupported Media type
+    Given path 'entities'
+    And request building
+    And header Content-Type = 'application/unkwnown'
+    When method post
+    Then status 415
+
+Scenario: Create Building with wrong Media type (application/json with JSON-LD payload)
+    Given path 'entities'
+    And request building
+    And header Content-Type = 'application/json'
+    When method post
+    Then status 400
+    And match response.detail == 'Request payload must not contain @context term for a request having an application/json content type'
```

Comment:
It's better to check on response.title as it is defined in the spec (detail is specific to the implementation)

Reply:
(eventually, you can check response.detail is not empty)